modules/drivers/druid-jdbc/test/metabase/driver/druid_jdbc/nested_field_columns_test.clj

```diff
@@ -15,6 +15,7 @@
 (set! *warn-on-reflection* true)
 
 (deftest ^:parallel json-details-only-test
+  (is (= 0 1) "the nested test gets invoked")
```

Comment:
shouldn't this fail? 😱

Reply:
Well spotted! I had doubts the tests actually run, so I added a couple of forced failures. As you see, they are not failing. Something to look into...